x-pack/plugins/ml/public/application/trained_models/models_management/models_list.tsx

```diff
@@ -470,6 +472,21 @@ export const ModelsList: FC<Props> = ({
             return !isPopulatedObject(item.pipelines);
           },
         },
+        {
+          name: i18n.translate('xpack.ml.inference.modelsList.testModelActionLabel', {
+            defaultMessage: 'Test model',
+          }),
+          description: i18n.translate('xpack.ml.inference.modelsList.testModelActionLabel', {
+            defaultMessage: 'Test model',
+          }),
+          icon: 'inputOutput',
+          type: 'icon',
+          isPrimary: true,
+          available: (item) => isTestable(item),
+          onClick: (model) => {
+            setShowTestFlyout(model);
+          },
```

Comment:
nit: 
```suggestion
          onClick: setShowTestFlyout,
```

Reply:
Updated in https://github.com/elastic/kibana/pull/128359/commits/2a9f47840a7e077a838e19e46673fad819b0f80c